src/test/kotlin/ar/edu/unahur/obj2/semillasAlViento/ParcelaTest.kt

```diff
@@ -0,0 +1,64 @@
+package ar.edu.unahur.obj2.semillasAlViento
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.assertions.throwables.shouldThrowAny
+import io.kotest.matchers.collections.shouldContain
+
+internal class ParcelaTest: DescribeSpec({
+    val parcela = Parcela(20, 1, 8)
+    describe ("Una parcela de 20m de ancho x 1m de largo") {
+
+        it("tiene una superficie de 20 m2") {
+            parcela.superficie().shouldBe(20)
+        }
+
+        it("soporta hasta 4 plantas") {
+            parcela.cantidadMaximaPlantas().shouldBe(4)
+        }
+
+        describe("se planta soja") {
+            for (num in 1..4) {
+                parcela.plantar(Soja(2019, 1.1f))
+            }
```

Comment:
Buena idea. También hay un `repeat`:
```suggestion
            repeat (4) {
                parcela.plantar(Soja(2019, 1.1f))
            }
```